thinc/compat.py

```diff
@@ -31,7 +31,10 @@
     import torch
 
     has_torch = True
-    has_torch_gpu = torch.cuda.device_count() != 0
+    has_torch_cuda_gpu = torch.cuda.device_count() != 0
+    has_torch_mps = hasattr(torch, "has_mps") and torch.has_mps
+    has_torch_mps_gpu = has_torch_mps and torch.backends.mps.is_available()
+    has_torch_gpu = has_torch_cuda_gpu or has_torch_mps_gpu
```

Comment:
Will this redefinition trip people up if they're using code that checks this boolean? Because we're editing some of our own tests to now use `has_torch_cuda_gpu` instead of `has_torch_gpu`, it feels like this could be semantically breaking?

Reply:
I wasn't sure about that either. On the one hand yes, on the other hand why would anybody use this? If I want to know if Torch thinks there are GPUs I would just ask Torch?

I am also wondering if we should underscore-prefix the new (since Thinc 8.1) `has_` variables to avoid these compatibility issues in the future?

Reply:
I have reverted this to `has_torch_gpu = has_torch_cuda_gpu`, since `has_torch_gpu = has_torch_cuda_gpu or has_torch_mps_gpu` is not needed for this PR. I think it's still something that we have to device before releasing 8.1.